packages/atomic/src/components/commerce/atomic-commerce-no-products/e2e/atomic-commerce-no-products.e2e.ts

```diff
@@ -0,0 +1,85 @@
+import {test, expect} from './fixture';
+
+test.describe('when there are results', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(
+      'http://localhost:4400/iframe.html?id=atomic-commerce-no-products--with-results&viewMode=story'
+    );
+  });
+
+  test('should not be visible', async ({noProducts}) => {
+    await expect(noProducts.ariaLive()).not.toBeVisible();
+  });
+
+  test.describe('after executing a search query that yields no results', () => {
+    test.beforeEach(async ({searchBox}) => {
+      await searchBox.hydrated.waitFor();
+      // eslint-disable-next-line @cspell/spellchecker
+      await searchBox.searchInput.fill('gahaiusdhgaiuewjfsf');
+      await searchBox.submitButton.click();
+    });
+
+    test('should have aria live', async ({noProducts}) => {
+      // eslint-disable-next-line @cspell/spellchecker
+      await expect(noProducts.ariaLive('gahaiusdhgaiuewjfsf')).toBeVisible();
```

Comment:
I dont understand the `aria-live` having a check on "visible" ? It's something that's hidden by definition.


Should it be instead a check that "aria-live has text value of xyz" ?

Reply:
:suspect: https://playwright.dev/docs/actionability#visible
I think the aria-live thingy should be an empty bounding box, but the Playwright doc lead me to believe the opposite 🤔